common/helpers.py

```diff
@@ -181,14 +192,18 @@ def get_ceph_osd_tree():
         return open(path, 'r').readlines()
 
 
-def get_ceph_versions():
+def get_ceph_versions(json_fmt=False):
     if DATA_ROOT == '/':
         output = subprocess.check_output(['ceph', 'versions'])
         return output.decode('UTF-8').splitlines()
 
     path = os.path.join(DATA_ROOT, "sos_commands/ceph/ceph_versions")
     if os.path.exists(path):
-        return open(path, 'r').readlines()
+        if json_fmt:
```

Comment:
Since we need this function to return the same whether it is called from file or real i think its best to the json bit in the caller. You'll also need to add simplejson to requirements.txt.